test/curator_test.rb

```diff
@@ -0,0 +1,188 @@
+  end
+
+  def test_it_can_add_an_artist
+    @curator.add_artist(@artist_1)
+
+    assert_equal [@artist_1], @curator.artists
+
+    @curator.add_artist(@artist_2)
+
+    assert_equal [@artist_1, @artist_2], @curator.artists
+  end
+
+  def test_it_can_find_artist_by_id
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+    @curator.add_artist(@artist_3)
+    expected = @curator.find_artist_by_id("1")
+
+    assert_equal expected, @artist_1
+
+    actual = @curator.find_artist_by_id("3")
+
+    assert_equal @artist_3, actual
+  end
+
+  def test_it_can_find_photograph_by_id
+    @curator.add_photograph(@photo_1)
+    @curator.add_photograph(@photo_2)
+    actual = @curator.find_photograph_by_id("2")
+
+    assert_equal @photo_2, actual
+  end
+
+  def test_it_can_find_photographs_by_artist
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+    @curator.add_artist(@artist_3)
+    @curator.add_photograph(@photo_1)
+    @curator.add_photograph(@photo_2)
+    @curator.add_photograph(@photo_3)
+    @curator.add_photograph(@photo_4)
+    expected = [@photo_3, @photo_4]
+    actual = @curator.find_photographs_by_artist(@artist_3)
+
+    assert_equal expected, actual
+  end
+
+  def test_it_can_find_artists_with_multiple_photographs
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+    @curator.add_artist(@artist_3)
+    @curator.add_photograph(@photo_1)
+    @curator.add_photograph(@photo_2)
+    @curator.add_photograph(@photo_3)
+    @curator.add_photograph(@photo_4)
+
+    actual = @curator.artists_with_multiple_photographs
+    expected = [@artist_3]
+
+    assert_equal expected, actual
+  end
+
+  def test_it_can_find_photographs_taken_by_artist_by_country
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+    @curator.add_artist(@artist_3)
+    @curator.add_photograph(@photo_1)
+    @curator.add_photograph(@photo_2)
+    @curator.add_photograph(@photo_3)
+    @curator.add_photograph(@photo_4)
+
+    actual = @curator.photographs_taken_by_artist_from("United States")
+    expected = [@photo_2, @photo_3, @photo_4]
+
+    assert_equal expected, actual
+
+    actual = @curator.photographs_taken_by_artist_from("Argentina")
+    assert_equal [], actual
+  end
+
+  def test_it_can_load_photographs_from_csv
+    @curator.load_photographs('./data/photographs.csv')
+    @curator.photographs.each do |photo|
+      assert_instance_of Photograph, photo
+    end
+  end
+
+  def test_it_can_load_artists_from_csv
+    @curator.load_artists('./data/artists.csv')
+    @curator.artists.each do |artist|
+      assert_instance_of Artist, artist
+    end
+  end
+
+  def test_it_can_find_photographs_taken_between_range
+    @curator.load_artists('./data/artists.csv')
+    @curator.load_photographs('./data/photographs.csv')
+    actual = @curator.photographs_taken_between(1950..1965)
+    actual.each do |photo|
+      assert (1950..1965).to_a.include? photo.year.to_i
+    end
```

Comment:
This is alriiiight, but you're effectively duplicating the code you needed here in the test. A more thorough test would be looking at WHICH photos you got back and do they meet your criteria